source/docs/a4.cargo,crates_and_basic_project_structure.md

```diff
@@ -3,21 +3,21 @@ title: Cargo, Crates and Basic Project Structure
 
 ## Cargo
 
-Cargo is Rust’s built-in Package Manager. But mainly it uses for,
+Cargo is Rust’s awesome built-in package manager. It is mainly used for:
```

Comment:
Related with https://github.com/learning-rust/site/issues/25 as well. I don't know! using "awesome" not very good but not bad either :)